app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="post-section">
+  <% content_for :title, "#{@user.name}'s Posts" %>
+  <%= render "users/user", user: @user %>
+  <% @user.posts.each do | post | %>
+    <%= link_to user_post_path(@user, post) do %>
+      <%= render "post", post: post %>
+
+      <div>
+        <h4>
+          <span>Comments</span>
+          <span>for Post: </span>
+          <span><%="#{post.title}"%> </span>
+        </h4>
+        <div class='postComments'>
+          <% post.recent_comments.each do |comment| %>
+            <div>
+              <p>
+                <span><strong><%= comment.user.name%>:</strong></span>
+                <span><%= comment.text %></span>
+              </p>
+            </div>
+            <%if can? :destroy, comment%>
+              <%= button_to "Delete Comment",
+               user_post_comment(comment.author_id,
+                comment.post_id, comment), 
+                method: :delete, data: {turbo: false}%>
+          <% end %>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
- [ ] When I try to create a post, I get this syntax error:

![image](https://github.com/Fahim-Khan-P/blog/assets/77106962/301d4daa-6fc6-48db-90df-c13b41b6caef)

The closing `% end %` tag for the opening `<% post.recent_comments.each do |comment| %>` is missing. Kindly add it to avoid getting this error. 👍 🖥️

Reply:
Thank you for your feedback, I apologize for missing that requirement in my previous response. You are correct, I will add the missing tag in the next commit!